src/api/routes/offer.js

```diff
@@ -58,6 +58,27 @@ module.exports = (app) => {
         }
     });
 
+    /**
+     * Gets all the offers of a certain company from the db
+     */
+    router.get("/company/:companyID", async (req, res, next) => {
```

Comment:
Throughout the project we are writing Id like so, I don't have a strong opinion about either version, I just want it to be consistent

```suggestion
    router.get("/company/:companyId", async (req, res, next) => {
```